XamarinCommunityToolkitSample/ViewModels/Behaviors/BehaviorsGalleryViewModel.cs

```diff
@@ -49,7 +49,11 @@ public class BehaviorsGalleryViewModel : BaseViewModel
 			new SectionModel(
 				typeof(ImpliedOrderGridBehaviorPage),
 				nameof(ImpliedOrderGridBehavior),
-				AppResources.AutoGridLayoutBehaviorDescription)
+				AppResources.AutoGridLayoutBehaviorDescription),
+			new SectionModel(
+				typeof(MaxLengthReachedBehaviorPage),
+				nameof(MaxLengthReachedBehavior),
+				AppResources.MaxLengthReachedBehaviorShortDescription)
```

Comment:
AppResources.MaxLengthReachedBehaviorShortDescription),

(please add a comma to the end of the line)